templates/blocks/review_item.liquid

```diff
@@ -0,0 +1,23 @@
+<div class="reviews-wrapper">
+  <div class="ratings-wrapper sc-flex-col sc-justify-center sc-align-items-center">
```

Comment:
`ratings-wrapper` is not following the BEM methodology. Please, change according to the parents name.